rotkehlchen/tests/unit/test_history_events.py

```diff
@@ -0,0 +1,51 @@
+from rotkehlchen.accounting.constants import EVENT_CATEGORY_MAPPINGS
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.base import (
+    HistoryEvent,
+    HistoryEventSubType,
+    HistoryEventType,
+)
+from rotkehlchen.constants.assets import A_ETH
+from rotkehlchen.fval import FVal
+from rotkehlchen.types import Location, TimestampMS
+
+
+def test_serialize_with_invalid_type_subtype():
+    """Test that serialize an event with invalid type/subtype does not raise exception"""
+    event_type = HistoryEventType.TRANSFER
+    event_subtype = HistoryEventSubType.SPEND
+    assert event_subtype not in EVENT_CATEGORY_MAPPINGS[event_type]
+    event = HistoryEvent(
+        event_identifier='1',
+        sequence_index=1,
+        timestamp=TimestampMS(1),
+        location=Location.KRAKEN,
+        event_type=event_type,
+        event_subtype=HistoryEventSubType.NONE,
+        asset=A_ETH,
+        balance=Balance(amount=FVal(1)),
+    )
+    event.event_subtype = event_subtype  # do here cause ctor will raise for invalid subtype
```

Comment:
I believe this is a typo? what is ctor? I believe you deletec constructor

Reply:
ctor is a common abbreviation for constructor